docs/source/modules/zos_backup_restore.rst

```diff
@@ -48,17 +48,21 @@ data_sets
 
   include
     When *operation=backup*, specifies a list of data sets or data set patterns to include in the backup.
+    When *operation=backup*, specifies a list of data sets or data set patterns to include in the backup.
```

Comment:
duplicate line in .rst

Reply:
regenerated the docs and looks good now

Reply:
I suspect the correction removes a diff so the file does not show up in the PR but is in in the branch as it should be so I am resolving this. 
https://raw.githubusercontent.com/ansible-collections/ibm_zos_core/cherry-picked-1110-beta-into-dev/docs/source/modules/zos_backup_restore.rst